src/App.css

```diff
@@ -1,38 +1,21 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+:root{
+  --fontSize-desktop-heading: 64px;
+  --fontSize-desktop-subHeading: 50px;
+  --fontSize-desktop-subSubHeading: 30px;
+  --fontSize-desktop-body: 20px;
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+  --fontSize-mobile-heading: 30px;
+  --fontSize-mobile-subHeading: 20px;
+  --fontSize-mobile-body: 16px;
```

Comment:
@praneershrest css vars are a really cool tool that lets you not hard declare magic numbers all over your CSS. so for example lets say 4 pages all have different components using font size 16px and then all the sudden the design is update to make them all larger to say 18px, we can change that here and everywhere else they just instantly update. So in components instead of using `font-size: 16px;` we can use `font-size: var(--fontSize-mobile-body);`